submissions/VasylRosokha/Popup/index.html

```diff
@@ -0,0 +1,123 @@
+                                </li>
+                                <li class="menu-item">
+                                    <a href="#" class="menu-item-link"><img src="img/icons/mail.png" alt="" class="menu-item-img">Mail</a>
+                                </li>
+                                <li class="menu-item">
+                                    <a href="#" class="menu-item-link"><img src="img/icons/calendar.png" alt="" class="menu-item-img">Calendar</a>
+                                </li>
+                                <li class="menu-item">
+                                    <a href="#" class="menu-item-link"><img src="img/icons/news.png" alt="" class="menu-item-img">News</a>
+                                </li>
+                                <li class="menu-item">
+                                    <a href="#" class="menu-item-link"><img src="img/icons/maps.png" alt="" class="menu-item-img">Map</a>
+                                </li>
+                                <li class="menu-item">
+                                    <a href="#" class="menu-item-link"><img src="img/icons/files.png" alt="" class="menu-item-img">Files</a>
+                                </li>
+                                <li class="menu-item">
+                                    <a href="#" class="menu-item-link"><img src="img/icons/contacts.png" alt="" class="menu-item-img">Contacts</a>
+                                </li>
+                                <li class="menu-item">
+                                    <a href="#" class="menu-item-link"><img src="img/icons/tanslation.png" alt="" class="menu-item-img">Translate</a>
+                                </li>
+                                <li class="menu-item">
+                                    <a href="#" class="menu-item-link"><img src="img/icons/videos.png" alt="" class="menu-item-img">Videos</a>
+                                </li>
+                                <li class="menu-item">
+                                    <a href="#" class="menu-item-link"><img src="img/icons/search.png" alt="" class="menu-item-img">Search</a>
+                                </li>
+                                <li class="menu-item">
+                                    <a href="#" class="menu-item-link"><img src="img/icons/photos.png" alt="" class="menu-item-img">Photo</a>
+                                </li>
+                                <li class="menu-item">
+                                    <a href="#" class="menu-item-link"><img src="img/icons/documents.png" alt="" class="menu-item-img">Document</a>
+                                </li>
+                            </ul>
+    
+                            <input type="checkbox" id="popup-more" class="more">
+                            <label for="popup-more" class="more-label">More</label>
```

Comment:
Logically when the second part of the popup is open this button should be hidden.